### PR TITLE
[10.0] [FIX] partner_email _check debug instead of error when logging

### PR DESCRIPTION
**Pull request by ntsirintanis**
Log a `debug` message instead of an `error` message when failing to import  `import validate_email`

### CODE REVIEW

**Review by hbrunn — APPROVED** [review]

File: partner_email_check/models/res_partner.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 try:
     from validate_email import validate_email
 except ImportError:
-    _logger.error('Cannot import "validate_email".')
+    _logger.debug('Cannot import "validate_email".')
```
Comment (pedrobaeza) [inline partner_email_check/models/res_partner.py:14]: Please put info as other import errors.

Comment (hbrunn) [reply partner_email_check/models/res_partner.py:14]: I asked my colleague to follow the conventions: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#importerror

Comment (pedrobaeza) [reply partner_email_check/models/res_partner.py:14]: OK, I miss the discussion that leads to this convention, as I would prefer info for seeing it always when starting my Odoo service. Any way, let's keep it as convention says.

Comment (hbrunn) [reply partner_email_check/models/res_partner.py:14]: thanks. I can't point to a discussion currently, but if I remember correctly the argument was that odoo's dependencies will disallow installing a module without the good libraries, and when debugging, you should turn on debug logging

Comment (pedrobaeza) [reply partner_email_check/models/res_partner.py:14]: Yeah, makes sense.

**Review by NL66278 — APPROVED** [review]
:+1: Approve although personally I also like info more... but let us follow conventions.

**Comment by hbrunn** [comment]
then let's add a pr to the conventions and have a discussion there

**Pull request closed without merging**